libraries/.foodcritic

```diff
@@ -0,0 +1,4 @@
+~FC011
```

Comment:
I think the real issue is that foodcritic thinks the libraries directory is a cookbook because of the `resources` directory within it. We can rename the directory to avoid these issues. See [this issue](https://github.com/acrmp/foodcritic/issues/148)